extensionsGallery-insider.json

```diff
@@ -5145,15 +5145,76 @@
 					],
 					"statistics": [],
 					"flags": "preview"
-				}
+				},
+				{
+					"extensionId": "101",
+					"extensionName": "ads-searcheverywhere",
+					"displayName": "Search Everywhere",
+					"shortDescription": "Search tables, views, stored procedures, functions by name or code.",
+					"publisher": {
+						"displayName": "MikhailT",
+						"publisherId": "MikhailT",
+						"publisherName": "MikhailT"
+					},
+					"versions": [
+						{
+							"version": "1.0.0",
+							"lastUpdated": "07/23/2023",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/MikhailProfile/SearchEverywhere/releases/tag/ads"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://github.com/MikhailProfile/SearchEverywhere/blob/main/png/search.png?raw=true"
```

Comment:
please update this to: https://raw.githubusercontent.com/MikhailProfile/SearchEverywhere/main/png/search.png

Reply:
Updated.